src/mapping_igvc/msg/Point2D.msg

```diff
@@ -0,0 +1,3 @@
+# A 2d point
```

Comment:
Would [`geometry_msgs/Pose2D`](http://docs.ros.org/api/geometry_msgs/html/msg/Pose2D.html) be appropriate as a substitute for this message?

Reply:
I considered it, but the meaning of a [`geometry_msgs/Pose2D`](http://docs.ros.org/api/geometry_msgs/html/msg/Pose2D.html) is different from just a 2D point, and since this ends up inside another custom message anyways (`ConeObstacle`), this doesn't really add any complexity to either usage or the build process.